nav2_behaviors/README.md

```diff
@@ -6,10 +6,26 @@ The package defines:
 - A `Recovery` template which is used as a base class to implement specific recovery timed action server - but not required.
 - The `BackUp`, `Spin` and `Wait` recoveries.
```

Comment:
It looks like you forgot to delete the recovery stuff that mirrors it on top :smile:

Reply:
Tarnation! Should be good now.